allensdk/brain_observatory/behavior/data_objects/metadata/ophys_experiment_metadata/ophys_project_code.py

```diff
@@ -0,0 +1,29 @@
+from allensdk.brain_observatory.behavior.data_objects.metadata.\
+    behavior_metadata.project_code import ProjectCode
+from allensdk.internal.api import PostgresQueryMixin
+
+
+class OphysProjectCode(ProjectCode):
+    """Unique identifier for the project this OphysExperiment is associated
+    with. Project ids can be used internally to extract a project code.
+
+    If the returned project id is null, return -1 for the id.
```

Comment:
Curious why you don't leave it as `None`

If you plan on keeping the logic to fill with a dummy value, please refactor setting the dummy value if missing into the base class constructor so that that logic does not need to be repeated here and the code is easier to follow.

Reply:
Fixed to just use a default the class.